addons/event_hr_skills/models/event_registration.py

```diff
@@ -0,0 +1,42 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models
+
+
+class EventRegistration(models.Model):
+    _inherit = 'event.registration'
+
+    def _create_resume_lines(self):
+        resume_line_vals_list = []
+        for registration in self:
+            if (
+                registration.state != 'done'
+                or not registration.event_id.tag_ids.category_id.resume_line_type_id
+            ):
+                continue
+
+            line_type = registration.event_id.tag_ids.category_id.resume_line_type_id[0]
+            resume_line_vals_list.extend([{
+                    'employee_id': employee.id,
+                    'event_id': registration.event_id.id,
+                    'name': registration.event_id.name,
+                    'date_start': registration.event_id.date_begin,
```

Comment:
When in doubt, order keys alphabetically, it helps finding the one we are looking for when trying to follow some variables through the code